package.json

```diff
@@ -1,42 +1,62 @@
 {
   "name": "taskcluster-client",
-  "version": "1.2.1",
+  "version": "2.0.0",
   "author": "Jonas Finnemann Jensen <jopsen@gmail.com>",
   "description": "Client for interfacing taskcluster components",
   "license": "MPL-2.0",
+  "files": [
+    "build",
+    "apis.json"
+  ],
+  "main": "build/index.js",
+  "browser": "build/browser.js",
   "scripts": {
-    "test": "./test/runtests.sh"
-  },
-  "browser": "./browser.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/taskcluster/taskcluster-client.git"
+    "build:browser": "webpack",
+    "build:node": "babel src --out-dir build --source-maps inline --ignore node_modules",
+    "build": "npm run build:node && npm run build:browser",
```

Comment:
I'm talking about consistency between all the libraries we have... We can't do each in it's own snowflake way... and we can copy/paste this much between all libraries... that's what babel-compile solves...

Look I'll admit webpack is probably nice for many reason, but we can't maintain so many libraries if all of them have config.